hor4tio/ft_containers
# Add push_back & pop_back in vector.hpp

- Add push_back() & pop_back() to vector.hpp
- Add vector_test.cpp to test push_back()
- Add vector_test.sh to compile, launch and verify leaks on vector_test.cpp on Linux & Mac
- Add GitHub workflows to start the script when someone commits or make a PR to the repo
